h2o-r/h2o-package/R/models.R

```diff
@@ -2029,6 +2029,21 @@ h2o.scoreHistory <- function(object) {
   }
 }
 
+#'
+#' Retrieve GLM Model Score History buried in GAM model
+#' @export
+h2o.scoreHistoryGAM <- function(object) {
+    o <- object
+    if( is(o, "H2OModel") ) {
+        sh <- o@model$glm_scoring_history
```

Comment:
why not just return( o@model$glm_scoring_history )? I dont understand the purpose of the if statement

Reply:
Yes, that should work.